src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -36,17 +33,38 @@
     <Compile Include="System\Net\NetworkInformation\IPv6InterfaceProperties.cs" />
     <Compile Include="System\Net\NetworkInformation\MulticastIPAddressInformation.cs" />
     <Compile Include="System\Net\NetworkInformation\MulticastIPAddressInformationCollection.cs" />
-    <Compile Include="System\Net\NetworkInformation\NetworkAddressChange.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkAvailabilityEventArgs.cs" />
     <Compile Include="System\Net\NetworkInformation\NetworkInterface.cs" />
     <Compile Include="System\Net\NetworkInformation\NetworkInterfaceComponent.cs" />
     <Compile Include="System\Net\NetworkInformation\nodetype.cs" />
     <Compile Include="System\Net\NetworkInformation\OperationalStatus.cs" />
     <Compile Include="System\Net\NetworkInformation\PhysicalAddress.cs" />
     <Compile Include="System\Net\NetworkInformation\PrefixOrigin.cs" />
-    <Compile Include="System\Net\NetworkInformation\SafeCancelMibChangeNotify.cs" />
-    <Compile Include="System\Net\NetworkInformation\SafeFreeMibTable.cs" />
     <Compile Include="System\Net\NetworkInformation\ScopeLevel.cs" />
     <Compile Include="System\Net\NetworkInformation\SuffixOrigin.cs" />
+    <Compile Include="System\Net\NetworkInformation\TcpConnection.cs" />
+    <Compile Include="System\Net\NetworkInformation\TcpState.cs" />
+    <Compile Include="System\Net\NetworkInformation\TcpStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\UdpStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\UnicastIPAddressInformation.cs" />
+    <Compile Include="System\Net\NetworkInformation\UnicastIPAddressInformationCollection.cs" />
+    <Compile Include="System\Net\NetworkInformation\InternalIPAddressCollection.cs" />
+    <Compile Include="$(CommonPath)\System\NotImplemented.cs">
```

Comment:
Nit: Please keep the files grouped under the comments (System.Net. common, Common, etc).